src/wordcloud2.js

```diff
@@ -371,8 +371,10 @@ if (!window.clearImmediate) {
     var getInfoGridFromMouseEvent = function getInfoGridFromMouseEvent(evt) {
```

Comment:
Would you be so kind and rename this function on the entire file to `getInfoGridFromMouseTouchEvent`?